doc/tutorials/setup_logical_partition_scenario_on_kbl_nuc_i7.rst

```diff
@@ -0,0 +1,154 @@
+.. _setup_logical_partition_scenario_on_kbl_nuc_i7:
+
+Guidelines of Setup Logical Partition Scenario on KBL NUC i7
+############################################################
+
+ACRN hypervisor supports logic partition scenario, in which the User OS (such as Clear Linux) running in a pre-launched VM can bypass the ACRN hypervisor and directly access isolated PCI devices. The following guidelines provide step by step instructions on how to set up the ACRN hypervisor logical partition scenario on KBL NUC i7 boards running two pre-launched VMs. The logical partition scenario on KBL NUC i7 is shown in :numref:`logical_partition_scenario_on_kbl_nuc_i7`.
+
+.. figure:: images/logical_partition_scenario_on_kbl_nuc_i7.png
+   :align: center
+   :width: 400px
+   :name: logical_partition_scenario_on_kbl_nuc_i7
+
+   The logical partition scenario on KBL NUC i7
+
+Prerequisites
+*************
+
+Disable "Intel Hyper Threading Technology" in the BIOS.
+
+In the logical partition scenario, two pre-launched VMs (running Clear Linux) are started by the ACRN hypervisor. To set up the Linux root filesystems for each VM, follow the Clear Linux OS bare metal installation guide to install Clear Linux OS on a SATA disk and a USB flash disk prior the setup, as the two pre-launched VMs will mount the root file systems via the SATA controller and the USB controller respectively.
+
+Update kernel image and modules of Pre-launched VM
+**************************************************
+
+Step1: On your development workstation, clone the ACRN kernel source tree, and build the Linux kernel image that will be used to boot the privileged VMs::
```

Comment:
Can you use a numbered list here? (see https://projectacrn.github.io/latest/developer-guides/doc_guidelines.html#lists)

Reply:
fix at doc v2 version